docs/public/components/creating-components.md

```diff
@@ -129,12 +167,45 @@ is to use a `Requires` and `Provides` structure to get and return value from you
         Conf config.Component
         Log  log.Component
     }
-
+    ```
+    
+!!! Info "Using other components"
+    You want to use another component within your own? Simply add it to the `Requires` struct and `Fx` will give it to
+    you at initialization ! Be careful of cycling dependencies though.
+    
+As for the output of our component we are going populate the `Provides` struct with the all the return values.
```

Comment:
I think 1 code example is better than 3. We should explain in 1 paragraph the `Provides` and `Requires` and then show a single example of code. Most people will only look at the code snippet and it will have everything they need.